rad/src/types/mod.rs

```diff
@@ -156,51 +190,138 @@ impl TryFrom<Value> for RadonTypes {
             Value::Map(_) => RadonMap::try_from(value).map(Into::into),
             Value::Text(_) => RadonString::try_from(value).map(Into::into),
             Value::Integer(_) => RadonInteger::try_from(value).map(Into::into),
-            _ => Ok(RadonBytes::from(value).into()),
+            Value::Bytes(_) => Ok(RadonTypes::from(RadonBytes::from(value))),
+            Value::Null => Err(RadError::Decode {
+                from: String::from("serde_cbor::Value::Null"),
+                to: String::from("RadonTypes"),
+            }),
+            Value::__Hidden => Err(RadError::Decode {
```

Comment:
I think you are not supposed to use that variant, and instead use a wildcard `_`. This allows for updates of the cbor library not breaking our builds.

```suggestion
            _ => Err(RadError::Decode {
```